motr/client.c

```diff
@@ -847,9 +847,9 @@ M0_INTERNAL int m0_calculate_md5_inc_context(
 
 	M0_PRE(pi != NULL);
 	M0_PRE(curr_context != NULL);
-	M0_PRE(bvec != NULL);
-	M0_PRE(bvec->ov_vec.v_count != NULL);
-	M0_PRE(bvec->ov_buf != NULL);
+	M0_PRE(ergo(bvec != NULL && bvec->ov_vec.v_nr != 0,
```

Comment:
if this means we dont assert when bvec->ov_vec.v_nr is 0 or bvec->ov_vec.v_count is 0 or bvec is 0 -- seems to be fine

Reply:
This means if bvec is NULL or bvec->ov_vec.v_nr == 0, in that case bvec->ov_vec.v_count can be NULL, bvec->ov_buf can be NULL.